pyshelf/search/utils.py

```diff
@@ -0,0 +1,32 @@
+from pyshelf.search.elasticsearch_wrapper import ElasticsearchWrapper
+
+
+def default_to_list(value):
+    """
+        Encapsulates non-list objects in a list for easy parsing.
```

Comment:
Encapsulates may not be a good word to use because programatically it usually means something else.

Maybe instead say that it ensures that a list is returned.  The rest of the functionality they can just look at.